examples/usecases/transformers-next-item-prediction.ipynb

```diff
@@ -0,0 +1,1234 @@
+{
```

Comment:
<p>Do we need the Loader? In previous Merlin Models, we could feed the nvt.Dataset object to model.fit and it will construct the loader automatically?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/853/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>you are absolutely right! turns out we don't need to be creating the loader, everything works perfectly fine if we just pass the dataset, removed this (and the creation of a loader for evaluation)</p>